src/lighter/datadog.py

```diff
@@ -0,0 +1,46 @@
+import logging, urllib2
+import lighter.util as util
+
+class Datadog(object):
+    def __init__(self, api_key, url=None):
+        self._api_key = api_key
+        self._url = url or 'https://app.datadoghq.com'
+        self._message_attribs = {
+            'aggregation_key': 'Lighter Deployment',
+            'source_type_name': 'lighter'
+        }
+
+    def notify(self, title, message, tags=[], priority='normal', alert_type='info'):
+        if self._api_key is None or self._api_key is '':
```

Comment:
No need to check `api_key != None` since `self._call()` does that later on anyway?